app/workers/education_form/education_facility.rb

```diff
@@ -65,11 +65,17 @@ def self.facility_for(region:)
     end
 
     # rubocop:disable Metrics/CyclomaticComplexity
-    def self.region_for(record)
+    def self.region_for(model)
+      record = model.open_struct_form
       # special case Philippines
-      return :western if (record.school&.address&.country || record.veteranAddress&.country) == 'PHL'
+      return :western if model.is_1990? && (record.school&.address&.country || record.veteranAddress&.country) == 'PHL'
```

Comment:
the name of the school hash might change, i can get rid of the conditionals and put in the extra fields later